readme.md

```diff
@@ -221,20 +221,6 @@ try {
 }
 ```
 
-### Graceful termination
-
-Using SIGTERM, and after 2 seconds, kill it with SIGKILL.
-
-```js
-const subprocess = execa('node');
-
-setTimeout(() => {
-	subprocess.kill('SIGTERM', {
-		forceKillAfterTimeout: 2000
-	});
-}, 1000);
-```
```

Comment:
The graceful exit behavior happens by default. It's probably not critical to add an example for it anymore.